### DISCUSSION TITLE
What is the capital of Brazil?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#303] [Galaxy Trivia] [answered]
**Hint:** not São Paulo 😊

**Comment by KelechiOdom10** [comment, marked as answer]
As the name suggests Brasília!

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/KelechiOdom10?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by peterbaccaro** [comment]
The current capital of Brazil, since its construction in 1960, is Brasilia. Rio de Janeiro was the country's capital between 1763 and 1960.

**Comment by thuongtruong1009** [comment]
The capital of Brazil is Brasília, which is located in the central-western region of the country. It was built specifically to be the capital and was inaugurated on April 21, 1960. Prior to that, the capital of Brazil was Rio de Janeiro.

**Comment by Mithun1508** [comment]
Brasília, city, federal capital of Brazil. It is located in the Federal District (Distrito Federal) carved out of Goiás state on the central plateau of Brazil. The current capital of Brazil, since its construction in 1960, is Brasilia
